go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -1121,12 +1120,14 @@ func flowsForDefaultBridge(bridge *bridgeConfiguration, extraIPs []net.IP) ([]st
 		if err != nil {
 			return nil, fmt.Errorf("unable to determine IPv4 physical IP of host: %v", err)
 		}
-		// table 0, SVC Hairpin from OVN destined to local host, DNAT and go to table 4
-		dftFlows = append(dftFlows,
-			fmt.Sprintf("cookie=%s, priority=500, in_port=%s, ip, ip_dst=%s, ip_src=%s,"+
-				"actions=ct(commit,zone=%d,nat(dst=%s),table=4)",
-				defaultOpenFlowCookie, ofPortPatch, config.Gateway.MasqueradeIPs.V4HostMasqueradeIP.String(), physicalIP.IP,
-				config.Default.HostMasqConntrackZone, physicalIP.IP))
+		for _, netConfig := range bridge.netConfig {
+			// table 0, SVC Hairpin from OVN destined to local host, DNAT and go to table 4
+			dftFlows = append(dftFlows,
+				fmt.Sprintf("cookie=%s, priority=500, in_port=%s, ip, ip_dst=%s, ip_src=%s,"+
+					"actions=ct(commit,zone=%d,nat(dst=%s),table=4)",
+					defaultOpenFlowCookie, netConfig.ofPortPatch, config.Gateway.MasqueradeIPs.V4HostMasqueradeIP.String(), physicalIP.IP,
```

Comment:
we should add a check here where we refuse to proceed if `netConfig.ofPortPatch` is empty.

Reply:
yeah have taken https://github.com/dceara/ovn-kubernetes/commit/2430febf2ba6141c9442c90fb48e94ed5e32fe6e from @dceara 
@dceara in EMEA hours tomorrow could you please clean this up for it to be prettier?

Reply:
Done, squashed into the 4ec76ceb1 "Add bridgeUDNConfiguration for UDN" commit.